libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -122,8 +123,8 @@ void AP_InertialSensor_Backend::_notify_new_gyro_raw_sample(uint8_t instance,
         uint64_t now = AP_HAL::micros64();
         struct log_GYRO pkt = {
             LOG_PACKET_HEADER_INIT((uint8_t)(LOG_GYR1_MSG+instance)),
-            time_us   : now,
-            sample_us : sample_us?sample_us:now,
+            time_us   : sample_us?sample_us:now,
```

Comment:
Unless I am missing something, SampleUS was just another call to AP_HAL::micros64() on the same thread and always about 19usec earlier than the call used to supply TimeUS. Hence it provides no additional information, but adds 8 bytes to a log message which is being generated at 1KHz. 
I don't know where SampleUS was being used, but the sequence counter adds information to the log which is useful in pymavlink/tools/mavfft_int.py